.github/workflows/native-custom_marian-mac.yml

```diff
@@ -0,0 +1,33 @@
+name: Native (Custom Marian) MacOS
```

Comment:
Remove file. This github check now asks all contributors to both with editing CMakeLists which has unnecessary paths. Keep a simpler mental model. Read Section 6.2 [here](https://www.cs.cmu.edu/~ckaestne/pdf/ecoop15.pdf).

Reply:
We want to see the reduced version works natively.  Resolving.